doc/CHANGES.txt

```diff
@@ -9,17 +9,37 @@ removed.  Use --error-test-filename and --regression-test-filename to set
 the file names; old command-line argument --junit-classname has been
 removed.
 
-New command-line argument --no-npe-contract disables the "No null pointer
-exceptions without null" heuristic contract.
-
-You can control which assertions appear in regression tests with new
-command-line arguments --no-regression-assertions and
---regression-assertions-about-exceptions; old command-line argument
+You can now indicate whether you want to view exceptions as expected, error or 
```

Comment:
"Whether you want to view" is vague; I don't think users will find this helpful.  Please avoid first person and also use more specific language.
I would start by saying that Randoop now splits its generated tests into three categories, and explain what they are and how they are output into tests.  Then you can say that the user can customize the categorization of a generated test into those three categories.